src/App.test.js

```diff
@@ -1,6 +1,281 @@
 import { render, screen, waitForElementToBeRemoved } from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
 import App from './App';
 
```

Comment:
Would prefer to see a large dataset like this loaded in as an import. Can store something like this in /data/data.js